.github/workflows/js-prs-issue.yml

```
@@ -14,8 +14,62 @@ jobs:
     if: (github.event_name == 'schedule' && github.repository_owner == 'silverstripe') || (github.event_name != 'schedule')
     runs-on: ubuntu-latest
     steps:
+      - name: Get Alerts List
+        id: get-alerts-list
+        run: |
+          ALERTS_LIST=''
+
+          # Get list of supported modules
+          # Assumes CMS 5 is the most recent stable version
+          curl -s -o __modules.json https://raw.githubusercontent.com/silverstripe/supported-modules/5/modules.json
+          if ! [[ $(jq -e '.' __modules.json) ]] >/dev/null 2>&1; then
```

```suggestion
          if ! [[ $(jq -e '.' __modules.json > /dev/null 2>&1) ]] ; then
```
Isn't this what we want?

Actually this is probably better, it's done elsewhere quite a bit when dealing with jq()

```
MODULES=$(jq -e '.' __modules.json) || true
if [[ $MODULES == "" ]]; then
```
Where `|| true` prevents github actions from halting because of the non zero exit code thrown by jq

---

Done. I don't think it's as clear what's happening there but I'm over it so I've made the change and added a comment about what it's doing.
Links in the description updated again with a new run on my fork.